src/ontology/mondo-edit.obo

```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
 synonym: "optic papilla edema" EXACT [MESH:D010211]
 synonym: "optic papilla oedema" EXACT OMO:0003005 []
 synonym: "optic papillitis" EXACT [MESH:D010211]
+synonym: "papilledema" EXACT [MONDO:0002003]
 synonym: "papilledema associated with decreased intraocular pressure" EXACT [MESH:D010211]
 synonym: "papilledema associated with increased intracranial pressure" EXACT [MESH:D010211]
 synonym: "papillitis" EXACT [DOID:10175, MESH:D010211]
 synonym: "papillitis, optic" EXACT [MESH:D010211]
 synonym: "papillitis, optic nerve" EXACT [MESH:D010211]
+synonym: "papilloedema" EXACT [NCIT:C3307]
 synonym: "retinal edema" RELATED [MESH:D010211]
 synonym: "retinal Edemas" RELATED [MESH:D010211]
 synonym: "retinal oedema" RELATED OMO:0003005 []
 xref: DOID:10175 {source="EFO:1001074", source="MONDO:equivalentTo"}
+xref: DOID:146 {source="MONDO:equivalentTo"}
 xref: EFO:1001074 {source="MONDO:equivalentTo", source="MONDO:EFO"}
 xref: ICD10CM:H35.81 {source="DOID:10175"}
 xref: ICD10CM:H46.0 {source="EFO:1001074", source="DOID:10175"}
+xref: ICD10CM:H46.0 {source="DOID:146"}
 xref: ICD10CM:H47.1 {source="DOID:10175"}
+xref: ICD10CM:H47.1 {source="DOID:146"}
 xref: ICD10CM:H47.10 {source="DOID:10175"}
+xref: ICD10CM:H47.10 {source="DOID:146"}
 xref: ICD10CM:H47.11 {source="DOID:10175"}
 xref: ICD9:362.83 {source="DOID:10175", source="MONDO:directSiblingOf"}
 xref: ICD9:377.0 {source="DOID:10175"}
+xref: ICD9:377.0 {source="DOID:146"}
 xref: ICD9:377.00 {source="MONDO:relatedTo", source="DOID:10175"}
+xref: ICD9:377.00 {source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:146"}
 xref: ICD9:377.01 {source="DOID:10175"}
+xref: ICD9:377.31 {source="MONDO:relatedTo", source="DOID:146"}
 xref: ICD9:377.31 {source="EFO:1001074", source="MONDO:equivalentTo", source="MONDO:i2s", source="DOID:10175"}
 xref: MedDRA:10030948 {source="EFO:1001074"}
+xref: MESH:D010211 {source="https://orcid.org/0000-0003-1967-3726", source="MONDO:equivalentTo", source="DOID:146"}
 xref: MESH:D010211 {source="EFO:1001074", source="DOID:10175"}
 xref: NCIT:C3307 {source="MONDO:otherHierarchy", source="DOID:10175"}
+xref: NCIT:C3307 {source="MONDO:equivalentTo", source="MONDO:otherHierarchy", source="MONDO:exact-label-match", source="DOID:146"}
 xref: SCTID:111526001 {source="DOID:10175"}
+xref: SCTID:111526001 {source="DOID:146"}
 xref: SCTID:155187009 {source="DOID:10175"}
+xref: SCTID:155187009 {source="DOID:146"}
 xref: SCTID:193426002 {source="DOID:10175"}
 xref: SCTID:194038006 {source="DOID:10175"}
+xref: SCTID:194038006 {source="DOID:146"}
 xref: SCTID:194041002 {source="DOID:10175"}
+xref: SCTID:194041002 {source="DOID:146"}
+xref: SCTID:248487006 {source="DOID:146"}
 xref: SCTID:248487006 {source="DOID:10175"}
 xref: SCTID:3170006 {source="DOID:10175"}
 xref: SCTID:423341008 {source="MONDO:relatedTo", source="DOID:10175"}
+xref: SCTID:423341008 {source="MONDO:equivalentTo", source="DOID:146"}
```

Comment:
duplicated x-ref.  Make sure you check which source is correct: equivalentTo or relatedTo.

Reply:
updated to `xref: SCTID:423341008 {source="MONDO:relatedTo", source="DOID:146"}`